settings.gradle

```diff
@@ -2,6 +2,8 @@ include ':annotations'
 include ':compiler'
 include ':compiler-api'
 include ':compiler-utils'
+include ':integration-tests:code-generator'
+include ':integration-tests:code-generator-tests'
```

Comment:
Any particular reason the contents of this module couldn't just go into `/integration-tests/code-generator/src/test/java`?

Reply:
Yes, because the main source set would need to be added to the Kotlin compiler plugin classpath. There's no separate classpath for the main and test source set.